associacao-brasileira-de-normas-tecnicas-ufrgs.csl

```diff
@@ -71,7 +85,7 @@
             <name-part name="family" text-case="uppercase"/>
             <name-part name="given" text-case="capitalize-first"/>
           </name>
-          <label form="short" text-case="capitalize-first" prefix=" (" suffix=".)"/>
+          <et-al font-style="italic"/>
```

Comment:
Removing the label here seems wrong -- you'd really not mark people as editors/translators?

Reply:
we are using editors and translators, but didn't understand what is wrong here.

Reply:
you're removing the label. The old version says (Org.) or (Orgs.) after their name. Yours doesn't.

Reply:
É porque a norma mudou e não muda mais o termo se single ou multiple

Reply:
But you're removing the label entirely. You've already changed the terms above so the label will never display in plural.

Reply:
Realmente estava faltando. Foi corrigido. Obrigada

Reply:
Só mais uma questão. Nos meus testes não aparece este texto "Tradução" a não ser que haja um tradutor.